### PR TITLE
Introduce index template for Juju.

### PR DESCRIPTION
**Pull request by frankban**

### CODE REVIEW

File: jujugui/templates/index.html.juju [hunk 1]
```diff
@@ -0,0 +1,421 @@
...
+      // Do not call _setAccount as we need to get the key from
+      // the config file which is not yet available.
+    </script>
+  </head>
+
+  <body>
+
+      <div class="hidden">
+        ## Include the svg sprite image inside the body.
+        <%include file="../static/gui/build/app/assets/stack/svg/sprite.css.svg" />
```
Comment (hatched) [inline jujugui/templates/index.html.juju]: This will need to be switched to be replaced by the go template instead of mako

**Comment by jujugui** [comment]
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/juju-gui/4347/
Test PASSed.

**Comment by jujugui** [comment]
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/juju-gui/4348/
Test PASSed.

**Comment by hatched** [comment]
:+1:  Looks good! We'll just have to make sure any modifications are done to both - probably a good idea to send an email.

**Comment by frankban** [comment]
:shipit:

**Comment by jujugui** [comment]
Status: merge request accepted. Url: http://ci.jujugui.org:8080/job/juju-gui-merge

**Comment by jujugui** [comment]
Build failed: Attempt to land pull request failed
        build url: http://ci.jujugui.org:8080/job/juju-gui-merge/1486

**Comment by frankban** [comment]
Looks spurious.
:shipit:

**Comment by jujugui** [comment]
Status: merge request accepted. Url: http://ci.jujugui.org:8080/job/juju-gui-merge

**Pull request merged**